cmake/modules/build_program_in_subdir.cmake

```diff
@@ -0,0 +1,17 @@
+# AUTHOR(S): Rashad Kanavath <rashad km gmail>
+# PURPOSE: 	 CMake macro to build a grass executable (program) under sub directory
+#            which is passed as argument to macro
+# COPYRIGHT: (C) 2000 by the GRASS Development Team
```

Comment:
Why is this file the only one of Rashad Kanavath with a 2000 copyright date instead of 2020?
```suggestion
# COPYRIGHT: (C) 2020 by the GRASS Development Team
```